Ellipsis Clarification (#349)
Pycord-Development/guide: docs/introduction.mdx

This list is not exhaustive, but it should give you a good idea of what you need to know to get started.

> When you see ellipsis(That's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!

Reviewer: ```suggestion
When you see ellipsis (that's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!
```

> When you see ellipsis(That's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!

Reviewer: ```suggestion
When you see ellipsis – that's the three dots! – in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!
```

> When you see ellipsis (that's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!

Reviewer: ```suggestion
When you see ellipsis – that's the three dots! – in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!
```

> When you see ellipsis (that's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!

Reviewer 2: ```suggestion
When you see an ellipsis (that's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be code for a command, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!
```
